docs/oidc_initiation.rst

```diff
@@ -21,14 +21,29 @@ to return an authentication request to the platform
     from lti_tool.views import OIDCLoginInitView
 
     urlpatterns = [
-        path("init/<uuid:registration_uuid/", OIDCLoginInitView.as_view(), name="init"),
-        ...
+        path("init/<uuid:registration_uuid/", OIDCLoginInitView.as_view(
+            main_msg=(
+                "Your browser prevents embedded content from using cookies.  To work "
+                "around this, the content must be opened in a new tab or window.  "
+            ),
+            click_msg="Open a new tab or window now.",
+            loading_msg="Loading..."
+        ), name="init"),
+        # ...
```

Comment:
Let's leave this example as it was, since it demonstrated the "as-is" use-case described in the language preceding.

An example demonstrating use of the new arguments can be shown in a separate example, alongside the description of the arguments and use case.

Reply:
OK.  I'll add a second example showing the new feature.

Reply:
Restored the previous example and added a new one.

Reply:
Looks great, thanks!